src/main/scala/no/ndla/audioapi/ComponentRegistry.scala

```diff
@@ -34,9 +49,21 @@ object ComponentRegistry
 
   ConnectionPool.singleton(new DataSourceConnectionPool(dataSource))
 
+  val amazonClient = new AmazonS3Client(new BasicAWSCredentials(AudioApiProperties.StorageAccessKey, AudioApiProperties.StorageSecretKey))
```

Comment:
Ja, det vil vi helt klart, men det er litt avhengig av at vi kommer over på ny arkitektur, der instansene startes med en instansprofil. Ingen av de serverene vi kjører nå har det.
Usikker på om det er verdt bryet med å ta ned og sette opp både test og staging environmentene for å starte de opp igjen med en instansprofil.